packages/kit/src/core/load_config/index.js

```diff
@@ -135,8 +135,13 @@ export function validate_config(config) {
 	// resolve paths
 	const { paths } = validated.kit;
 
-	if (paths.base !== '' && !paths.base.startsWith('/')) {
-		throw new Error('config.kit.paths.base must be a root-relative path');
+	if (paths.base !== '') {
+		if (!paths.base.startsWith('/')) {
+			throw new Error('config.kit.paths.base must be a root-relative path');
+		}
+		if (paths.base.endsWith('/')) {
+			throw new Error('config.kit.paths.base must not end with /');
```

Comment:
I wonder if rather than throwing an error we should just drop the trailing `/`

Reply:
Isn't it better that the user is conscious of this as it's a one time config?